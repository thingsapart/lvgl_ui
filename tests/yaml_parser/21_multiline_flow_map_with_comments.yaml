config: {
  key: value, # comment 1
  num: 123,
  # comment 2
  valid: true
}
