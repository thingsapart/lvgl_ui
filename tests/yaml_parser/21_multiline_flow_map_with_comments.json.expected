[{
		"config":	{
			"key":	"value",
			"num":	123,
			"valid":	true
		}
	}]
